packages/frontend/src/components/Trade/Short/index.tsx

```diff
@@ -841,7 +841,7 @@ const CloseShort: React.FC<SellType> = ({ open }) => {
       amount.isGreaterThan(0) &&
       shortVaults.length &&
       amount.lt(finalShortAmount) &&
-      neededCollat.isLessThan(MIN_COLLATERAL_AMOUNT)
```

Comment:
Wondering if it Is the neededCollat wrong or it is not loading properly causing this issue?

Reply:
Hmmm It seems to load properly. Let me know if l'm missing anything or if it's still not working for you.